.github/workflows/lint_self_test.yml

```diff
@@ -0,0 +1,37 @@
+name: linter self test
+
+on:
+  push:
+    branches-ignore:
+      # ignore tmp branches used by bors
+      - 'staging.tmp*'
+      - 'trying.tmp*'
+      - 'staging*.tmp'
+      # ignore nolints
+      - 'nolints'
+      # do not build lean-x.y.z branch used by leanpkg
+      - 'lean-3.*'
+      # ignore staging branch used by bors, this is handled by bors.yml
+      - 'staging'
```

Comment:
From what I can tell, `bors.yml` doesn't run the lint self test (anymore?). 

It's probably exceedingly unlikely given how rarely the style linter is edited, but it's possible some conflict in a batch could cause breakage, right?
```suggestion
```

Reply:
In case you didn't notice, this is a different file.  Bors has never looked at this check (i.e. would have merged a batch even if this check failed).  There's an explicit list of required checks in bors.toml.  I've added staging to the ignore list here because this check spawns 6 jobs, and costs us precious ci time.

If you think bors should require this check as well, then we can do this in another PR (but I'm not a big fan).  In that case, the staging line here must indeed be removed.

Reply:
Ah, I see. Thanks for the explanation!